src/index.d.ts

```diff
@@ -878,6 +878,11 @@ declare namespace admin.remoteConfig {
      * ETag of the current Remote Config template (readonly).
      */
     readonly etag: string;
+
+    /**
+     * Version information of the current Remote Config template.
```

Comment:
Does this return just a number?  

I lean toward "information for" unless it's just a "version number of"

Reply:
Thanks! It is not just a number. Changed to `information for`.